src/cgi-bin/share.php

```diff
@@ -0,0 +1,147 @@
+<?php
```

Comment:
Can we move that to `src/php`? The build script could just copy it to the root..?

Reply:
Sure. What do you think about adding the whole thing as a submodule?